.gitignore

```diff
@@ -103,3 +105,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# Baobab slurm script
+job.sh
```

Comment:
pep8: a file should end with a newline

Reply:
it was a file I am using on a cluster machine.